### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by elyor-doniyorov**
We used HTML and CSS, CSS media queries to build the project.
We made the design responsive to all screen sizes.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,1056 @@
...
+
+    .footer-menu-links {
+      margin-left: 10px;
+      width: 100%;
+    }
+
+    .footer-menu-links a {
+      padding: 10px;
+      margin: 0;
+    }
```
Comment (andresporras3423) [inline styles.css]: Group selectors with the same style definition

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,1056 @@
...
+  
+      .nav1-top,
+      .nav-sticky,
+      .date {
+        display: none;
+      }
+  
+      .logo-div {
+        display: none;
+      }
```
Comment (andresporras3423) [inline styles.css]: Group selectors with the same style definition

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,1056 @@
...
+        width: 100%;
+      }
+
+      .box-wrapper {
+        width: 100%;
+        padding: 0 0;
+      }
+      .latest-box {
+        width: 100%;
+      }
```
Comment (andresporras3423) [inline styles.css]: selectors on lines 889 and 897 have the same style definition. Group selectors with the same style definition.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Priscilla Oni Olajumoke Priscilla & Elyor Doniyorov Doniyor o`g`li** 
Good job with your first submission for this project :muscle:  :muscle: 
There are some issues to correct before you can go ahead:
- [ ] Main navigation as subnavigation bar looks broken between 769px and 1200px:

- [ ] In large screen size, "Latests news" section should have 4 articles per row
- [ ] In medium screen size, "latest funding" section should have 3 articles per row
- [ ] In medium screen size, "Categories" section should have 2 categories per row 
- [ ] In Medium screen size, "Latests news" section should have 3 articles per row
- [ ] Your footer should cover all the width of the page:

- [ ] In your CSS file, group selectors with the same style definition
- [ ] Make sure your project pass all the linter checks
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle:

### Hi @EL28DEV and @prolajumokeoni     ! :wave: :wave:

#### You have done well so far :clap: However, there are changes required for you to make in order for this project to be approved. Kindly work on them and submit them for another code review.

**General**

- [ ] To help with improving this project and to make look as similar as possible to the actual site. I recommend using the following Chrome extensions: 
- [Font Picker](https://chrome.google.com/webstore/detail/font-picker/mmjbimgpcbaegjiieojddickpjbdkeej): it helps determine the font-family, size, line height and other significant properties of the font used in the actual site. 
- [ColorZilla](https://chrome.google.com/webstore/detail/colorzilla/bhlhnicpbhignbdhedgjhgdocnmhomnp?hl=en) : Will help pick the colors used off of the page.

**Recommended**

- **Large Screens**

- [ ] The navigation bar needs more work done on it. The width is larger than the one in the [TNW ](https://thenextweb.com/) site. Please compare and fix it.
- [ ] You have a horizontal overflow. Fix it.
- [ ] As I stated earlier, use the Chrome extension tools I recommended to determine the font-size, family, and other properties, in order to have your site looking as similar as possible to the actual one.
- [ ] There should be some spacing between the 3 main articles and the "Latest Deals"Sections.
- [ ] The latest funding section, needs a little bit more work done, use the tools I recommended.
- [ ] Remember to add spacing between the sections, they're all sticking together.
- [ ] The category section should have bullet points and lines, please refer to the TNW site.

image

- [ ] Please fix the footer to look like the one in the actual site.

- **Medium Screens**

- [ ] Most of the page breaks between 769px and 1200px, the issue is likely from the navigation bar. Fix it.

- **Small Screens**

- [ ]  On small screens, the page is tilted entirely to the left, it has something to do with your navigation bar as well. so fix it.

__That's it team! ✋ __
__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Comment by elyor-doniyorov** [comment]
We appreciate your reviews. Hopefully the last commit will be approved by you. Then we can start to prepare for the next project. Thanks)

**Review by Georjane — CHANGES_REQUESTED** [review]
## Status: Changes Required 🔁 🚫
### Hi Team 🙌
Very nice work you have done on the TNW website clone. Your efforts can clearly be seen in your work. You have done a great job so far. 
- [ ] Please kindly make sure all your linter checks are passing in order for this project to be approved.

Feel free to reach out to me on Slack if you have any questions, Witah Georjane. 

Goodluck on your project and Happy Coding 💻

**Pull request closed without merging**